app/controllers/CancelDeclarationController.scala

```diff
@@ -51,53 +52,67 @@ class CancelDeclarationController @Inject() (
 )(implicit ec: ExecutionContext)
     extends FrontendController(mcc) with I18nSupport with Logging with WithUnsafeDefaultFormBinding {
 
-  def displayPage: Action[AnyContent] = (authenticate andThen verifyEmail).async { implicit request =>
+  def displayPage(adt: String): Action[AnyContent] = (authenticate andThen verifyEmail).async { implicit request =>
```

Comment:
Not sure what 'adt' stands for. Can we expand this to a more meaningful name please

Reply:
'adt' is short for additionalDeclarationType. It's used elsewhere in the code but perhaps the lack of context here makes it vague. I could:
1- Add a comment to clarify what it stands for
2- Use the full form i.e. additionalDeclarationType
3- Use a more meaningful form such as addDecType

Which do you prefer?

Reply:
2,  personally i would use the name 'additionalDecType' :slightly_smiling_face:

Reply:
I went for `additionalDecType` throughout 👍